addons/knowledge/models/knowledge_article.py

```diff
@@ -1333,23 +1333,26 @@ def _get_article_member_permissions(self, additional_fields=False):
         sql = f'''
     WITH article_permission as (
         WITH RECURSIVE article_perms as (
-            SELECT a.id, a.parent_id, m.id as member_id, m.partner_id,
-                   m.permission
+            SELECT a.id, a.parent_id, a.is_desynchronized, m.id as member_id,
+                   m.partner_id, m.permission
               FROM knowledge_article a
          LEFT JOIN knowledge_article_member m
                 ON a.id = m.article_id
         ), article_rec as (
             SELECT perms1.id, perms1.id as article_id, perms1.parent_id,
                    perms1.member_id, perms1.partner_id, perms1.permission,
-                   perms1.id as origin_id, 0 as level
+                   perms1.id as origin_id, 0 as level,
+                   perms1.is_desynchronized as article_is_desynchronized
```

Comment:
In any case this deserves its own testing methods as well (tests that don't work before the changes and that work after the changes).

As again it's very touchy and complicated.
You will already find a lot of tests about the permission stuff, I let you find a good place to add your own.

Reply:
The reason it's different is because I did not know it was there. There is no reason to be different though, so I will change it to make it consistent